docs/guides/overrides-exporter.md

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
Wasn't really sure of where this should live or what docs are expected to look like. The exporter seemed big enough that it should probably have _some_ documentation

Reply:
This doc looks solid. Good work